include/rapidcheck/detail/BitStream.hpp

```diff
@@ -40,6 +40,7 @@ template <typename Source>
 template <typename T>
 T BitStream<Source>::next(int nbits, std::false_type) {
   using SourceType = decltype(m_source.next());
+  SourceType sourceTypeLength = (sizeof(SourceType) * 8);
```

Comment:
Ah, I missed `numBits<T>()`. I'll update this function to use that rather than this assignment.